StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1516UnitTests.cs

```diff
@@ -372,6 +372,38 @@ public class Foo
             await this.VerifyCSharpFixAsync(testCode, fixedCode);
         }
 
+        [Fact]
+        public async Task TestThatCodeFixWorksOnFieldsAdjacentToMultiLineFields()
+        {
+            string testCode = @"using System;
+
+public class Foo
+{
+    private string experiment =
```

Comment:
I just checked. It does report for that code. Whats weird is the location the diagnostic is at. It reports for char 1 at that line, so thats a difference. but I think the location we report is better.